test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -1900,7 +1909,8 @@ def test_cgroup_reserve_mem(self):
             self.logger.info('Vmem-2: %s' % vmem2.value)
             vmem_resv = vmem1 - vmem2
             self.logger.info('Vmem resv: %s' % vmem_resv.value)
-            self.assertEqual(vmem_resv.value, 97280)
+            # self.assertEqual(vmem_resv.value, 97280)
```

Comment:
Did you mean to take this commented line out?

Reply:
I did. I opened this draft PR for Alexis. It was the easiest way to provide him a diff. I will be closing this once he updates his copy.